src/main/steps/c100-rebuild/child-details/navigationController.ts

```diff
@@ -7,12 +7,17 @@ import {
   C100_CHILDERN_DETAILS_PARENTIAL_RESPONSIBILITY,
   C100_CHILDERN_DETAILS_PERSONAL_DETAILS,
   C100_CHILDERN_FURTHER_INFORMATION,
+  C100_CHILDERN_OTHER_CHILDREN_NAMES,
```

Comment:
Have a separate navigation controller for other children

Reply:
sometimes having separate files with the same information fails on sonarqube because of duplicate code @vivek-sekhar

Reply:
But yes will make these changes also

Reply:
> sometimes having separate files with the same information fails on sonarqube because of duplicate code @vivek-sekhar

This would not be duplicate as the nav controller will just have code definition to its need.

Reply:
okay completed